deployer/grafana/central_grafana.py

```diff
@@ -0,0 +1,161 @@
+"""
+Ensures that the central grafana at https://grafana.pilot.2i2c.cloud
+is configured to use as datasource the authenticated prometheus instances
+of all the clusters that we run.
+"""
```

Comment:
This is not new content! It's the same code, with only some of the comments updates.
I believe this is why GitHub got fooled to believe it's a new file and not one that got moved 🙄

Reply:
Yeah the comment updates will always make git thinks it's a new file in diffs, unfortunately. From the object database point of view, this is a completely new object, and that's why it behaves in this way.